plane_wave.ipynb

```diff
@@ -0,0 +1,643 @@
+{
```

Comment:
That's obviously a matter of taste, but I prefer kebab-case over snake_case for file names, unless it's not technically possible.

Either way, it would be good to be consistent.

Reply:
An old bad habit presumably originating from my assembler coding days :-) Changed for consistency to plane-wave.ipynb, which I then also have done for greens-functions.ipynb (I hope there is no dependency on this in the other notebooks).